docker/checkout.bash

```diff
@@ -54,6 +54,8 @@ fi
 # NOTE: The ibeo_msgs package is ignored because on build the cmake files in that package run a sed command 
 #       which can make them incompatible with a new ros version after a source switch
 git clone https://github.com/astuff/astuff_sensor_msgs 
+#rosbridge_suite is a ROS meta-package including all the rosbridge packages.
+git clone https://github.com/RobotWebTools/rosbridge_suite.git
```

Comment:
Looks like ros2 is the default branch, but it would be nice to specify the branch here in case that changes. Also since we are using a third part package, best to make sure we use a verified stable commit